internal/engine/experiment/riseupvpn/riseupvpn.go

```diff
@@ -107,10 +111,9 @@ func (tk *TestKeys) AddGatewayConnectTestKeys(v urlgetter.MultiOutput, transport
 			tk.FailingGateways = append(tk.FailingGateways, *gatewayConnection)
 		}
 	}
-	return
```

Comment:
`staticcheck` warning

Reply:
(The Go plugin for VSCode now includes `staticcheck` so I suddenly see all these warnings.)

Reply:
Obviously I need to update my vscode... Thanks for the hint

Reply:
My pleasure!